rosetta/validator/validator.go

```diff
@@ -66,23 +62,6 @@ func (v *Validator) Network(network identifier.Network) error {
 // => https://github.com/optakt/flow-dps/issues/51
 func (v *Validator) Block(block identifier.Block) error {
 
-	blockID, err := flow.HexStringToIdentifier(block.Hash)
```

Comment:
Why remove this validation?

Reply:
Because we no longer have access to this information with the changed interfaces. There is an issue for it.